core/src/main/res/values/styles.xml

```diff
@@ -60,4 +60,8 @@
     <item name="cardBackgroundColor">@color/card_view_background</item>
   </style>
 
+  <style name="progressbar_style" parent="ThemeOverlay.AppCompat">
```

Comment:
should be named like `ThemeOverlay.ProgressBar` and you can just set `parent=""`